packages/react-components/src/molecules/__tests__/ToastCard.test.tsx

```diff
@@ -12,11 +13,39 @@ it('does not render a toast when no message is provided', () => {
   const { queryByTitle } = render(<ToastCard>content</ToastCard>);
   expect(queryByTitle('Alert')).not.toBeInTheDocument();
 });
+it('renders a toast card with a link', () => {
+  const { getByRole } = render(
+    <ToastCard toastContent={<Link href="http://example.com">test</Link>}>
+      content
+    </ToastCard>,
+  );
+  expect(getByRole('link')).toHaveAttribute('href', 'http://example.com');
+});
 
 it('renders a toast card with alert', () => {
   const { getByTitle, getByText } = render(
-    <ToastCard toastText="Warning">content</ToastCard>,
+    <ToastCard toastContent="Warning">content</ToastCard>,
   );
   expect(getByTitle('Alert')).toBeInTheDocument();
   expect(getByText('Warning')).toBeVisible();
 });
+
+it('renders a toast card with paper clip icon', () => {
+  const { getByTitle, getByText } = render(
+    <ToastCard type="attachment" toastContent="explanation text">
+      content
+    </ToastCard>,
+  );
+  expect(getByTitle('Paper Clip')).toBeInTheDocument();
+  expect(getByText('explanation text')).toBeVisible();
+});
+
+it('renders a toast card with a clock icon', () => {
+  const { getByTitle, getByText } = render(
+    <ToastCard type="live" toastContent="explanation text">
+      content
+    </ToastCard>,
+  );
+  expect(getByTitle('Clock')).toBeInTheDocument();
+  expect(getByText('explanation text')).toBeVisible();
+});
```

Comment:
The props as currently defined allow a combination of passing a type, but not passing children. The behavior then (icon is also not shown) should be tested